resolwe_bio/tests/processes/test_reads_filtering.py

```diff
@@ -105,3 +105,34 @@ def test_sortmerna_paired(self):
                           [{'url': 'fastqc/rRNA_rew_filtered_fastqc/fastqc_report.html',
                             'refs': ['fastqc/rRNA_rew_filtered_fastqc'],
                             'name': 'View'}])
+
+    def test_trimmomatic_single(self):
+        reads = self.prepare_reads()
+        inputs = {'reads': reads.pk,
+                  'trailing': 3,
+                  'crop': 5}
+        filtered_reads = self.run_processor('trimmomatic-single', inputs)
+
+        self.assertFiles(filtered_reads, 'fastq', ['filtered_reads_trimmomatic_single.fastq.gz'], compression='gzip')
+        self.assertFields(filtered_reads, "fastqc_url", [{'url': 'fastqc/reads_fastqc/fastqc_report.html',
```

Comment:
Just a cosmetic comment, please replace double quotes with single quotes. We use double quotes for strings meant for human consumption and single quotes for everything else.